### ISSUE TITLE
data source for training depth model

### ISSUE THREAD

**Issue by kl456123** [mileyan/pseudo_lidar#2]
I think may be you use more data to train depth model , especially the data from val set.

**Comment by mileyan** [comment]
In this paper, we only use the training set to train PSMNet, without any data on the validation set. I will release the model soon.
